library/include/internal/hipsolver-types.h

```diff
@@ -14,8 +14,7 @@
 #endif
 
 #include <hip/hip_complex.h>
-#include <hip/hip_runtime_api.h>
-#include <hip/library_types.h>
+#include <hip/hip_runtime.h>
```

Comment:
Why include `hip/hip_runtime.h`? Isn't that primarily for HIP language code?

Reply:
I was attempting to fix the compile failure on cuda11. I figured CUDART_VERSION was not defined and `hip/hip_runtime.h` would include it, but this doesn't seem to have fixed the problem.

Reply:
If this is not needed, could we leave it as-is?

Reply:
I can try reverting it, sure. I'm unclear on which header defines CUDART_VERSION, so it's possible I might have to change it back to how it is now.